### PR TITLE
Clues for admins to edit user profiles, accounts

### PR DESCRIPTION
**Pull request by weedySeaDragon**
#### PT URL: https://www.pivotaltracker.com/story/show/169760526

Provide clues so admins can edit user information: make links to editting user profiles and user accounts more obvious.

- I've put in some visual cues and words.  Are they helpful?  Suggestions?  See the screenshots below.
- I picked an icon for the user profile and for the user account.  Are these ok? 
- I don't really know what to use for the user account.  A _user account_ shows  membership + application + payment status).  I didn't see a FontAwesome icon that represents those well.
- We don't have to use a FontAwesome icon; we can use any image we want.  I just picked from FontAwesome to keep it easy. (We could use a tiny version of on of the SHF graphics if it'd show up well enough.)

## Changes proposed in this pull request:
1.  Small changes to UI to help provide clues. (See screenshots below)

### 1. ~~Added words to column headers to explain links;~~ added tooltips to each entry
Reverted the headers.  (Tooltips remain)

_~~Added words to 2 column headers to explain where you'll go when you click on an entry.~~ (And also added tooltips to each entry so when you hover you'll see something like "Edit account for user Firstname Lastname"_

allUsers-headers-reverted

---

### 2. Edit Profile page: Added icon in header to go to the _view_ account
_Added an icon to the Header that links to the (view) account page._

edit-profile-icon-to-view-acct

---

### 3(a). View Account page: Added **_edit_** icon (the pencil thing) in header to go to the _edit_ account page.  Also has the icon to go to the Edit Profile page.
_Added an "edit" icon to the Header that links to the edit account page._

view-acct-icons-edit-profile

---

### 3. Edit Account page: Added icon in header to go to the edit profile page.
_Added an icon to the Header that links to the edit profile page._

edit-acct-icon-to-profile

---
### 4.  User Account: Application section - add links, show all companies

- Show all companies for an application. Link each company #, name to the company
- Linked the application state to the application. 
- Used i18n for the "User Info" section title

many-companies-links-AdminTitle-localed

---

---
## Ready for review:
@thesuss

### CODE REVIEW

File: config/locales/sv.yml [hunk 1]
```diff
@@ -893,6 +895,8 @@ sv:
       not_sent: inte skickat
       edit_member_status: &edit_member_status Redigera medlemsstatus
       pay_membership: &pay_membership Betala medlemsavgift
+      edit_user_profile: &edit_user_profile Edit the profile for %{name}
+      admin_user_info_title: Användare Info
```
Comment (thesuss) [inline config/locales/sv.yml]: Information om användaren

File: config/locales/sv.yml [hunk 2]
```diff
@@ -936,10 +942,16 @@ sv:
       applications_tooltip: Användaren har en öppen ansökan
       expire_date: *expire_date
       member_packet: *member_packet
```
Comment (thesuss) [inline config/locales/sv.yml:958]: pkt

File: config/locales/sv.yml [hunk 3]
```diff
@@ -936,10 +942,16 @@ sv:
...
+      edit_profile: Edit Profile
```
Comment (thesuss) [inline config/locales/sv.yml:959]: not needed

File: config/locales/sv.yml [hunk 4]
```diff
@@ -936,10 +942,16 @@ sv:
...
+      edit_account: (Edit Acct.)
```
Comment (thesuss) [inline config/locales/sv.yml:960]: not needed

**Comment by thesuss** [comment]
The icon by the very red header makes no sense... 
Use a button at the bottom like for company or application? 

As for the user table, short info, short headline. 
As for the swedish version that was as it was :)

File: config/locales/sv.yml (hunk 2 above)
Comment (weedySeaDragon) [reply config/locales/sv.yml:958]: the "*" means that it's a reference to the actual entry.  (It's defined on line 881.  The "&" in _that_ entry means that it is the key definition.
In the `sv file it is "pkt"  .  But... in the `en` file it was "member packet".   I fixed it.
Good eye!

**Comment by weedySeaDragon** [comment]
_@thesuss said:_
> The icon by the very red header makes no sense...
> Use a button at the bottom like for company or application?

I thought it might be a quick way for the admin to switch back and forth between the User Account and User Profile for the user. 
I think maybe the icons could be helpful.  I popped them in there with the header because I couldn't quickly come up with a different place, and just wanted some feedback.

I also added an Edit icon so that if you (as an admin) are _viewing_ a user profile, you can click on the "edit" icon and quickly switch to the _edit_ view.  
I'll update the screenshots so you can see.  And if you don't like the icons at all, no worries.  We can take them out.

(And let me know what color to change the "very red header" to and I'll make it so. :-)

> As for the user table, short info, short headline.
> As for the swedish version that was as it was :)
- fixed.

**Pull request merged**